_includes/footer.html

```diff
@@ -35,9 +35,14 @@
                               <h2>Concur Blog</h2>
                             </div>
                             <p class="md-margin-bottom-40">
-                              <a href="https://www.concur.com/blog/" target="_blank" class="blog-link">
+                              <a href="{{ site.baseurl }}/blog/" class="blog-link">
                                 Read the latest news for apis, features, and best practices of the Concur Platform.
                               </a>
+                              <br>
+                              <a href="{{ site.posts[0].url }}">{{ site.posts[0].title}}</a>
+                              <br>
+                              <a href="{{ site.posts[1].url }}">{{ site.posts[1].title}}</a>
+                              <a></a>
```

Comment:
I think we should removed the first a tag ("Read the latest news for apis..."), and just show the 3 most recent blogs (2 for now since that's all we have).

Also, add the blog-link class to the other a tags so color is white instead of blue.